tests/unit/test_database.py

```diff
@@ -265,12 +267,109 @@ def test_db_cli(self):
         sys.argv = orig_argv
 
     def test_init_db(self):
-        DCPQueryDBManager().init_db(dry_run=True)
-        DCPQueryDBManager().init_db()  # dry_run is True by default
+        init_db(dry_run=True)
+        init_db()  # dry_run is True by default
 
     def test_drop_db(self):
-        DCPQueryDBManager().drop_db(dry_run=True)
-        DCPQueryDBManager().drop_db()  # dry_run is True by default
+        drop_db(dry_run=True)
+        drop_db()  # dry_run is True by default
+
+
+class TestDBManager:
+    process_file_link_ignore_duplicate_rule_sql = """
+        CREATE OR REPLACE RULE process_file_join_table_ignore_duplicate_inserts AS
```

Comment:
It would be ideal to increase DRY by importing these rules from a source that's common to the migration file and the test file, but I wouldn't worry too much about that right now (could file a note to do it in the future though).

Reply:
created a ticket